front-end/src/styles/App.css

```diff
@@ -17,3 +17,75 @@
 .App-header > * {
   padding: 0rem;
 }
+
+input[type="checkbox"] {
+  appearance: none;
+}
+
+input[type="checkbox"]::before {
+  content: "";
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  border: 2px solid crimson;
+  border-radius: 3px;
+  margin-right: 8px;
+}
+
+input[type="checkbox"]:checked::before {
+  content: '\2713';  
+  display: block; 
+  text-align: center;
+  font-size: 16px; 
+  line-height: 16px; 
+  color: white; 
+  background-color: crimson;
+}
+
+input[type="checkbox"]:hover::before {
+  border-color: #555;
+}
+
+input[type="checkbox"]:focus::before {
+  box-shadow: 0 0 3px #555;
+}
+
+select {
+  appearance: none; 
+  padding: 3px;
+  border: 1px solid #555;
+  border-radius: 4px;
+  background-color: white;
+  width: 200px; 
+}
+
+
+button {
+  padding: 7px 7px;
+  font-size: 0.8rem;
+  font-weight: bold;
+  border-radius: 5px;
+  border: none;
+  outline: none;
+  background: crimson;
+  color: white;
+  cursor: pointer;
+  max-width: 100%; 
+  overflow: hidden; 
+  white-space: nowrap; 
+}
+
+button:hover {
+  background-color: rgb(238, 48, 86);
+}
+
+select {
+  width: 200px;
+  padding: 5px;
+  font-size: 12px;
+}
+
+select option:checked {
+  background-color: crimson; 
+  color: #fff; 
+}
```

Comment:
Similar styling options are inside modalStyles.css, fine to be here but should be compared and compiled. No need to have both. Also ensure all styling here are global changes